meilisearch/src/routes/indexes/documents.rs

```diff
@@ -341,7 +341,7 @@ pub async fn update_documents(
     let params = params.into_inner();
     debug!(parameters = ?params, "Update documents");
 
-    analytics.update_documents(&params, index_scheduler.index(&index_uid).is_err(), &req);
+    analytics.update_documents(&params, index_scheduler.index_exists(&index_uid) == Ok(true), &req);
```

Comment:
```suggestion
    analytics.update_documents(&params, index_scheduler.index_exists(&index_uid).map_or(false, |x| x), &req);
```

Reply:
Yeah, That would work but I think we have to pass the negation of the response meaning 
if the index exist we have to pass false to the function, Correct me if I am wrong.

Reply:
Ho! Yeah, you are right. It's exactly that, my bad 😅

Reply:
@Kerollmops 
Addressed the proposals and updated the PR.
Let me know of any issues